packages/sdk-js/src/index.ts

```diff
@@ -7,14 +7,17 @@ import {
   Attester,
   Verifier,
 } from '@kiltprotocol/actors-api'
-import * as types from '@kiltprotocol/types'
+import {
+  BlockchainUtils,
+  SubscriptionPromise,
+} from '@kiltprotocol/chain-helpers'
+import * as ChainHelpers from '@kiltprotocol/chain-helpers'
 
 export * from '@kiltprotocol/types'
 export * from '@kiltprotocol/core'
 export { Actors }
 
 export default {
-  types,
```

Comment:
These can't be accessed via the default export. It is an issue on the typescript side. The types can still  be accessed though, since they are all reexported directly (line 16)

Reply:
didn't see that, you're right